courses/B.yaml

```diff
@@ -4,6 +4,8 @@
     becomes unusable.  There are good and bad ways to do this.
   reading: |
     `Aalto scicomp data docs <https://scicomp.aalto.fi/data/>`__
```

Comment:
I guess it's worth just going ahead and removing this... it'll probably never be good enough.